### PR TITLE
Fixed french translation

### PR DESCRIPTION
**Pull request by CiramorDev**
I fixed the few mistakes in the correction. I hope it will be useful!

### CODE REVIEW

File: translations/base-fr.yaml
Comment (DerFeldspatz) [inline translations/base-fr.yaml:14]: grafik
Since it's rendered like this on Steam page the colon is not needed (neither is it used in English or other languages)

File: translations/base-fr.yaml
Comment (DerFeldspatz) [inline translations/base-fr.yaml:24]: Same here with the steam page title

**Review by DerFeldspatz — CHANGES_REQUESTED** [review]
The colons on the steam page titles are not needed. Syntax is fine 👍

**Comment by tobspr** [comment]
See above :)

**Comment by CiramorDev** [comment]
I think the file is good, just waiting on your approval @tobspr 
I will let you handle it from now (except if you need me to do something on my side?)

**Review by DerFeldspatz — APPROVED** [review]

File: translations/base-fr.yaml
Comment (krop) [inline translations/base-fr.yaml:261]: terminé

File: translations/base-fr.yaml
Comment (krop) [inline translations/base-fr.yaml:464]: Si elles ne peuvent pas être **assemblées**

File: translations/base-fr.yaml
Comment (krop) [inline translations/base-fr.yaml:592]: virtuelle

File: translations/base-fr.yaml
Comment (krop) [inline translations/base-fr.yaml:613]: a été **débloquée**

File: translations/base-fr.yaml
Comment (krop) [inline translations/base-fr.yaml:614]: de couleur

File: translations/base-fr.yaml
Comment (krop) [inline translations/base-fr.yaml:615]: pour les **peindre**

File: translations/base-fr.yaml
Comment (krop) [inline translations/base-fr.yaml:626]: les deux entrées sont **assemblées**

File: translations/base-fr.yaml
Comment (krop) [inline translations/base-fr.yaml:718]: **Afficheur** was correct

**Review by krop — CHANGES_REQUESTED** [review]

File: translations/base-fr.yaml
Comment (CiramorDev) [reply translations/base-fr.yaml:718]: I think affichage sounds better, but both work. What do you think Krop?

Comment (CiramorDev) [reply translations/base-fr.yaml:718]: Also, thanks for proofreading!

Comment (krop) [reply translations/base-fr.yaml:718]: Nope, `Affichage` has a completely different definition. AFAICS, these strings are about an electronic device to display information, hence `afficheur`. 
Quoting Wikipedia[1]:
« en électronique, un afficheur numérique (ou digital) est un dispositif comportant un ou plusieurs emplacements où l'on peut représenter un chiffre, ou une lettre alphabétique. »

vs. affichage[2]:
« technique de communication en extérieur qui consiste à installer et parfois gérer dans un espace public du mobilier urbain destiné à recevoir un support de nature promotionnelle. »

[1] https://fr.wikipedia.org/wiki/Afficheur
[2] https://fr.wikipedia.org/wiki/Affichage

Comment (CiramorDev) [reply translations/base-fr.yaml:718]: ok, bah je remet afficheur du coup

File: translations/base-fr.yaml
Comment (krop) [inline translations/base-fr.yaml:482]: same issue with the number here. The english string says « Colors the shapes on the left inputs... », so « Peint les formes venant de gauche... »

File: translations/base-fr.yaml
Comment (CiramorDev) [reply translations/base-fr.yaml:271]: Yes, but It makes more sense to have it be this way I think because it shows the stock you have right now and not the all-time stock. Also for the others it just sounds better and same, it is showing what is happening rn

File: translations/base-fr.yaml
Comment (CiramorDev) [reply translations/base-fr.yaml:422]: Yeah, it isn't, but it isn't wrong, so why not

File: translations/base-fr.yaml
Comment (CiramorDev) [reply translations/base-fr.yaml:464]: Oh yeah, that sounds way better, good idea

File: translations/base-fr.yaml
Comment (CiramorDev) [reply translations/base-fr.yaml:482]: I don't see the problem here

**Comment by CiramorDev** [comment]
I think we can say this is ready to be implemented @tobspr.

**Comment by tobspr** [comment]
Alright, thanks!

**Pull request merged**
